vendor/engines/journal/lib/weby/components/newsletter/assets/stylesheets/newsletter.css.scss

```diff
@@ -0,0 +1,8 @@
+.newsletter_component {
+  max-width: 300px;
+
+  input#email{
```

Comment:
Avoid using id selectors<br>Avoid qualifying id selectors with an element.<br>Opening curly brace `{` should be preceded by one space